doc/modules/cassandra/pages/auto-repair/overview.adoc

```diff
@@ -0,0 +1,125 @@
+= Overview of Auto Repair
+:navtitle: Auto Repair overview
+:description: Auto Repair concepts - How it works, how to configure it, and more.
+:keywords: CEP-37
+
+Repair orchestration inside Apache Cassandra. This fully Automated Repair scheduler inside Cassandra does not depend on the control plane, significantly reducing our operational overhead.
```

Comment:
Just a suggestion to make this more clear for users:

```suggestion
Auto Repair is a fully automated scheduler that provides repair orchestration inside of Apache Cassandra.

This significantly reduces operational overhead by removing the need for operators to deploy tooling to submit and manage repairs.
```

Reply:
Done